jira/client.py

```diff
@@ -4327,7 +4327,7 @@ def create_project(
 
         ps_list: List[Dict[str, Any]]
 
-        if permissionScheme is None:
+        if permissionScheme is None and not self._is_cloud:
```

Comment:
```suggestion
        if permissionScheme is None and self._is_cloud:
            # Let Jira decide the default field due to issues in handling the field
            # https://community.atlassian.com/t5/Jira-questions/Using-Default-Permission-Scheme-with-Create-Project-API-causes/qaq-p/1862489
        elif permissionScheme is None:
```

If we do want this change I would suggest doing it this way, so we can better document the reasoning for handling it in this way.